CHANGELOG.rst

```diff
@@ -6,19 +6,25 @@ Changelog
 
 Features:
 
-* Clean up code for schema hooks (:issue:`814`). Thanks :user:`taion`.
-* Minor performance improvement from simplifying ``utils.get_value`` (:issue:`811`). Thanks again :user:`taion`.
-* Add ``require_tld`` argument to ``fields.URL`` (:issue:`749`). Thanks
+- Clean up code for schema hooks (:issue:`814`). Thanks :user:`taion`.
+- Minor performance improvement from simplifying ``utils.get_value`` (:issue:`811`). Thanks again :user:`taion`.
+- Add ``require_tld`` argument to ``fields.URL`` (:issue:`749`). Thanks
   :user:`DenerKup` for reporting and thanks :user:`surik00` for the PR.
-* ``fields.UUID`` deserializes ``bytes`` strings using ``UUID(bytes=b'...')`` (:issue:`625`).
+- ``fields.UUID`` deserializes ``bytes`` strings using ``UUID(bytes=b'...')`` (:issue:`625`).
   Thanks :user:`JeffBerger` for the suggestion and the PR.
 
 Bug fixes:
 
-* Fields nested within ``Dict`` correctly inherit context from their
+- Fields nested within ``Dict`` correctly inherit context from their
   parent schema (:issue:`820`). Thanks :user:`RosanneZe` for reporting
   and :user:`deckar01` for the PR.
-* Includes bug fix from 2.15.3.
+- Includes bug fix from 2.15.3.
+
+Other changes:
+
+- *Backwards-incompatible*: Pre/Post-processors must return modified data.
```

Comment:
This should be in 3.0.0b12. I'll fix this.

Reply:
Right. I think it got screwed up during the rebase and I didn't notice.